tests/api/runtimes/test_nuclio.py

```diff
@@ -923,6 +923,34 @@ def test_deploy_with_node_selection(self, db: Session, client: TestClient):
             tolerations=tolerations,
         )
 
+    def test_compile_function_config_node_selector_enriched_from_project(
```

Comment:
do you want to test subsets (only project / only config) or is it already tested in UTs for `merge_dicts_with_precedence` ?

Reply:
In my opinion it is already UTs for `merge_dicts_with_precedence`, but maybe I will add a case for not having any NS defined to make sure nothing is broken.